liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/InstallFeatureMojo.java

```diff
@@ -47,12 +82,262 @@ protected void doExecute() throws Exception {
         if (skip) {
             return;
         }
-        
+
         // for liberty-assembly integration
         if (features == null) {
             return;
         }
+
+        checkServerHomeExists();
+
+        List<String> configurationFeatures = new ArrayList<String>();
+        List<Feature> configurationEsas = new ArrayList<Feature>();
+        populateConfigurationFeaturesLists(configurationFeatures, configurationEsas);
+
+        if (handleFallbackScenarios(configurationEsas)) {
+            return;
+        }
+
+        Set<File> downloadedJsons = downloadProductJsons();
+        if (downloadedJsons.isEmpty()) {
+            handleJsonsNotFound();
+            return;
+        }
+
+        List<String> dependencyFeatures = getDependencyFeaturesList();
+        List<String> serverFeatures = getServerFeaturesList(configurationFeatures.isEmpty());
```

Comment:
Can we change the term `configurationFeatures` to `pomFeatures` or something throughout the class? The word "configuration" is pretty generic.

Reply:
Changed to pluginListedFeatures